ssc/cmod_pvwattsv5.cpp

```diff
@@ -671,7 +672,7 @@ class cm_pvwattsv5_1ts : public cm_pvwattsv5_base
 		double wspd = as_double("wspd");
 		double alb = as_double("alb");
 
-		double shad_beam = 1.0;
+		double shad_beam = 1.0 - as_double("shaded_percent") / 100.0;
```

Comment:
I just want to make sure I'm not getting things turned around here. Does this look right?

Reply:
Yes!